src/Bicep.Core.Samples/Files/baselines/AKS_LF/main.diagnostics.bicep

```diff
@@ -52,7 +52,9 @@ resource aks 'Microsoft.ContainerService/managedClusters@2020-03-01' = {
         }
         servicePrincipalProfile: {
             clientId: servicePrincipalClientId
+//@[22:46) [BCP417 (Info)] The supplied value has been marked as secure but is being assigned to a target that is not expecting sensitive data. (bicep https://aka.ms/bicep/core-diagnostics#BCP417) |servicePrincipalClientId|
             secret: servicePrincipalClientSecret
```

Comment:
There are a couple of cases where I expected the RP to mark something as sensitive in the Swagger, but they didn't. Besides this target, the other big surprises were `resourceInput<'Microsoft.Compute/virtualMachines@2019-12-01'>.properties.osProfile.adminPassword` (annotated as a regular string) and `resourceInput<'Microsoft.KeyVault/vaults/secrets@2019-09-01'>.properties.value` (ditto). I didn't check all API versions for those.